core/templates/dev/head/exploration_editor/main_tab/StateResponses.js

```diff
@@ -284,20 +286,26 @@ oppia.controller('StateResponses', [
       trainingDataService.initializeTrainingData(
         explorationId, currentStateName);
     };
+
     $scope.suppressDefaultAnswerGroupWarnings = function() {
       var interactionId = $scope.getCurrentInteractionId();
+      var answerGroups = responsesService.getAnswerGroups();
+      var handledAnswersArray = [];
+      var choiceIndices = [];
+      var answerChoices = [];
+      var choiceRules = [];
+      var customizationArgs = stateCustomizationArgsService.savedMemento;
```

Comment:
No you are still not getting it correct. Please take a look at how this service is implemented in `EditorServices.js` file.
Check what is returns when you use `savedMemento`. As I said problem is not where you use variable. Problem is you didn't initialize the service by making a call to `stateCustomizationArgsService.init()` function. Once you will check how this service is implemented and what it returns for `savedMemento` you will know where exactly you are getting it incorrect.

I hope this is clear. If not then please ask.
Thanks